plugins/modules/system/filesystem.py

```diff
@@ -144,6 +164,22 @@ def create(self, opts, dev):
             cmd = "%s %s %s '%s'" % (mkfs, self.MKFS_FORCE_FLAGS, opts, dev)
         self.module.run_command(cmd, check_rc=True)
 
+    def wipefs(self, dev):
+        if platform.system() == 'FreeBSD':
+            msg = "module param state=absent is currently not supported on this OS (FreeBSD)."
+            self.module.exit_json(msg=msg)
```

Comment:
Shouldn't this bee `fail_json`? Otherwise users might miss the message and think the module did its job.

Reply:
I don't know what is best... I think it should be a transitionnal version, and broken things should be fixed (at least for a consistent behaviour between target OSes). So I did it: `fail_json` wins.